_includes/header.html

```diff
@@ -1,7 +1,20 @@
 <header class="site-header">
-  <img src="/images/uxjourneylogo.png" id="mylogo" href="https://myuxjourney.github.io/" alt="UX Journey Logo" style="max-width: 300px; max-height: 68vw; padding-left: 30px; padding-top: 2%;">
+  <img
+    src="/images/uxjourneylogo.png"
+    id="mylogo"
+    href="https://myuxjourney.github.io/"
+    alt="UX Journey Logo"
+    style="
+      max-width: 300px;
+      max-height: 68vw;
```

Comment:
This is me being extremely picky, but alphabetically, max-height would come before max-width (I'm so sorry haha I know this is absurd)